Buen-Aire-Frontend-App/screens/purpleair.js

```diff
@@ -43,6 +41,21 @@ class PurpleAirMap extends React.Component {
         );
     }
 
+    getPinColor(marker){
+        let pmValue = Number(marker['pm_2.5']);
+        var color = 'green';
+        if(pmValue > 50 && pmValue < 101) {
+            color = 'yellow';
+        }
+        if(pmValue < 100 && pmValue < 151) {
+            color = 'orange';
+        }
+        if(pmValue > 150) {
+            color = 'red';
+        }
+        return color;
```

Comment:
These ranges look a bit off. I think it should be:

- `pmValue >= 50 && pmValue < 100`
- `pmValue >= 100 && pmValue < 150`
- `pmValue >= 150`